docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+.. _connecting-crm:
+
+Getting Started on customer integration
+#######################################
+
+This getting started manual assumes a vendor sells services to its customers.
+The vendor manages the presence of those customers on the Nuts network through the Nuts registry.
+It's very likely the vendor has software to manage customer environments. We'll call it a *CRM* where the *customers* part relates to organizations and not people.
+This does not mean that a stand-alone installation isn't supported. In that case the vendor and organization are the same.
+
+The Nuts registry enables service discovery for organizations. The registry identifies organizations through their :ref:`DID <did>`.
+The DIDs are new and do not match the internal identifiers known to the CRM.
```

Comment:
"New" and "internal identifiers" is a bit vague imo. "Connection" makes me think of network connections, instead of mapping something from an external domain. Why not make it explicit, something like:

> The DIDs are unique, random identifiers which are generated when the organization is registered in the Nuts registry. After creation of the DID the CRM should store and map it to its customer record, so it can refer to it when updating the customer's DID Document and issue Verifiable Credentials.